src/Error/ExceptionRenderer.php

```diff
@@ -114,9 +125,10 @@ protected function _unwrap($exception)
      */
     protected function _getController()
     {
-        if (!$request = Router::getRequest(true)) {
+        if (!$request = $this->request ?: Router::getRequest(true)) {
```

Comment:
It's better to avoid variable assignment inside `if`. So please move it above and just do `if ($request === null)` here.

Reply:
Agreed. Wrote it like that to keep close to original code. Will rewite.